### DISCUSSION TITLE
Dynamic batch editing

### DISCUSSION THREAD

**Discussion by ghost** [martinrotter/rssguard#1214] [Q&A] [answered]
I recently saw that we can now batch edit feeds (<3) and I was wondering, is it possible to do it dynamically to partially replace the content?

For example, convert `server1.net/A` and `server1.net/B` to `server2.net/A` and `server2.net/B`

**Comment by martinrotter** [comment, marked as answer]
I was thinking about something like this, yes. It is not yet possible but will add it for next version. Will be likely regex based.

**Reply by ghost** [reply]
Good to know. Thanks for your work!

**Comment by thebestcloudislocal** [comment]
What happens if I open `database.db` in sqlitebrowser and enter `UPDATE Feeds SET source = replace(source, 'https://nitter.cz/', 'https://nitter.poast.org/');`? Can I click on "File | Write changes" or will RSS Guard be unhappy?
